modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/actions/PreIssueAccessTokenClientCredentialsGrantTestCase.java

```diff
@@ -0,0 +1,354 @@
+
+        String[] scopes = jwtClaims.getStringClaim("scope").split("\\s+");
+
+        Assert.assertTrue(ArrayUtils.contains(scopes, NEW_SCOPE_1));
+        Assert.assertTrue(ArrayUtils.contains(scopes, NEW_SCOPE_2));
+        Assert.assertTrue(ArrayUtils.contains(scopes, NEW_SCOPE_3));
+        Assert.assertTrue(ArrayUtils.contains(scopes, NEW_SCOPE_4));
+        Assert.assertFalse(ArrayUtils.contains(scopes, CUSTOM_SCOPE_3));
+        Assert.assertFalse(ArrayUtils.contains(scopes, CUSTOM_SCOPE_2));
+    }
+
+    @Test(groups = "wso2.is", description = "Verify the presence of the updated aud claims in the access token")
+    public void testTokenAUDClaimOperations() throws Exception {
+
+        String[] audValueArray = jwtClaims.getStringArrayClaim("aud");
+
+        Assert.assertTrue(ArrayUtils.contains(audValueArray, "zzz1.com"));
+        Assert.assertTrue(ArrayUtils.contains(audValueArray, "zzz2.com"));
+        Assert.assertTrue(ArrayUtils.contains(audValueArray, "zzz3.com"));
+        Assert.assertTrue(ArrayUtils.contains(audValueArray, "zzzR.com"));
+        Assert.assertFalse(ArrayUtils.contains(audValueArray, clientId));
+    }
+
+    @Test(groups = "wso2.is", description = "Verify the presence of the specified custom string claim in the access " +
+            "token")
+    public void testTokenStringClaimAddOperation() throws Exception {
+
+        String claimStr = jwtClaims.getStringClaim("custom_claim_string_1");
+        Assert.assertEquals(claimStr, "testCustomClaim1");
+
+    }
+
+    @Test(groups = "wso2.is", description = "Verify the presence of the specified custom number claim in the access " +
+            "token")
+    public void testTokenNumberClaimAddOperation() throws Exception {
+
+        Number claimValue = jwtClaims.getIntegerClaim("custom_claim_number_1");
+        Assert.assertEquals(claimValue, 78);
+
+    }
+
+    @Test(groups = "wso2.is", description = "Verify the presence of the specified custom boolean claim in the access " +
+            "token")
+    public void testTokenBooleanClaimAddOperation() throws Exception {
+
+        Boolean claimValue = jwtClaims.getBooleanClaim("custom_claim_boolean_1");
+        Assert.assertTrue(claimValue);
+    }
+
+    @Test(groups = "wso2.is", description = "Verify the presence of the specified custom string array claim in the " +
+            "access token")
+    public void testTokenStringArrayClaimAddOperation()
+            throws Exception {
+
+        String[] claimArray1 = {"TestCustomClaim1", "TestCustomClaim2", "TestCustomClaim3"};
+
+        String[] claimArray = jwtClaims.getStringArrayClaim("custom_claim_string_array_1");
+        Assert.assertEquals(claimArray, claimArray1);
+    }
+
+    @Test(groups = "wso2.is", description = "Verify the replacement of the 'expires_in' claim in the access token")
+    public void testTokenExpiresInClaimReplaceOperation() throws Exception {
+
+        if (jwtClaims.getClaim("expires_in") != null) {
```

Comment:
```suggestion
        if (jwtClaims.getClaim("exp") != null) {
```
`expires_in` comes in the token response body

Reply:
Updated with the commit 3f3b2671fc4aac10f5292af1606f409f6b30b2e9